.travis.yml

```diff
@@ -38,7 +38,7 @@ script:
   - tox -p all
   # Try to generate dist package. We should in fact try to do a complete build,
   # but that's larger task. For now this should do.
-  - ./build.sh dist
+  - bash -x ./build.sh dist
```

Comment:
`./build.sh prep` sometimes fails as well, please make that run with bash -x as well.

Reply:
done in #36